express_relay/sdk/python/express_relay/searcher/examples/simple_searcher_svm.py

```diff
@@ -70,20 +101,11 @@ async def bid_status_callback(self, bid_status_update: BidStatusUpdate):
                 result_details = f", transaction {result}"
         logger.info(f"Bid status for bid {id}: {bid_status.value}{result_details}")
 
-    async def bid_on_new_orders(self):
-        orders = await self.limo_client.get_all_orders_state_and_address_with_filters(
-            []
-        )
-        orders = [
-            order for order in orders if order["state"].remaining_input_amount > 0
-        ]
-        if len(orders) == 0:
-            logger.info("No orders to bid on")
-            return
-        for order in orders:
-            await self.evaluate_order(order)
-
-    async def evaluate_order(self, order: OrderStateAndAddress):
+    async def assess_opportunity(self, opp: Opportunity) -> BidSvm:
```

Comment:
same

Reply:
I will move this check to the other function so that this one only accepts OpportunitySvm